docs/ai/run/cli.md

```diff
@@ -19,8 +20,16 @@ Options:
           [string] [default: "https://unauthipfs.subquery.network/ipfs/api/v0/"]
       --ipfsAccessToken  A bearer authentication token to be used with the ipfs
                          endpoint                                       [string]
-  -h, --host             The ollama RPC host
+      --cacheDir         The location to cache data from ipfs. Default is a temp
+                         directory                                      [string]
+      --debug            Enable debug logging         [boolean] [default: false]
+      --logFmt           Set the logger format
+                        [string] [choices: "json", "pretty"] [default: "pretty"]
+  -h, --host             The LLM RPC host. If the project model uses ChatGPT
```

Comment:
```suggestion
  -h, --host             The LLM RPC host for non-OpenAI models. If the project model uses a OpenAI
```

Reply:
You can still set the host for OpenAI models, it just wont use the default value, instead it will use the default OpenAI endpoint